public/index.html

```diff
@@ -49,7 +49,7 @@
       <p class="justifyP">ADD STUFF</p><br>
       <img id="about-me-pic" src="assets/about-me.jpg" alt="">
       <p class="justifyP">ADD STUFF</p><br>
-      <p class="justifyP">ADD STUFF</p>
+      <p class="justifyP">ADD STUFF</p><br><br><br><br>
```

Comment:
**TODO FOR LOST POINTS**

Kabob-case your HTML attributes, i.e. ``justify-p``.